MAINTAINERS.md

```diff
@@ -6,5 +22,91 @@
 | Peter Nied | [peternied](https://github.com/peternied) | Amazon |
 | Barani Bikshandi | [bbarani](https://github.com/bbarani) | Amazon |
 | Sayali Gaikawad | [gaiksaya](https://github.com/gaiksaya) | Amazon |
+| Abhinav Gupta | [abhinavGupta16](https://github.com/abhinavGupta16) | Amazon |
+| Zelin Hao | [zelinh](https://github.com/zelinh) | Amazon |
+| Elaine | [zylzulu](https://github.com/zylzulu) | Amazon |
+
+[This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) explains what maintainers do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Zenhub Process Workflow
+We follow agile methodologies for our development and release process. We use zenhub to manage and track our stories and issues to effectively manage them over the sprint.
+
+### What is [Zenhub](https://www.zenhub.com/)?
+
+ZenHub is an agile project management and product roadmaps solution, natively integrated into GitHub. It is free to use for opensource repository and comes with a paid membership to manage private repositories. 
+We currently use zenhub only with our public and opensource repositories.
```

Comment:
Seems like a ZenHub sales pitch, lets focus on 'how' we use ZenHub in this document.  We could have a 'why' we picked ZenHub section that might go into those details.

Reply:
I am not I fully understand. Isn't the whole document about how to use zenhub? For example, how the pipelines work, creating issues with Zenhub, working on tasks etc.

Reply:
> Isn't the whole document about how we use zenhub? 

Yes, we do not need to educate anyone else on why we use it, we use it!  We can make another section for that, but that isn't very important until we consider using something else.